server.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] I recommend avoiding the use of `eslint disable` commands in your projects, as these errors and warnings are typically helpful for identifying areas where you can improve your code. By resolving these issues, you can write more clean and readable code that adheres to industry standards, which can make it easier to maintain your codebase over time. :100:

Reply:
Hi @NahnahAJ
Thank you for your feedback. However, I don't think I could make the corrections on this one because I was told to download Webpack differently than what was instructed a few weeks ago. see the instruction - 'webpack' should be listed in the project's dependencies, not devDependencies.

Reply:
Hello @unorjikingsley Alright then you can leave that one. I hope you have been able to fix the others

Reply:
thank you @NahnahAJ for the leniency. I wouldn't mind assistance cause I am stuck with another error after fixing yours.  😪

Reply:
What is it about @unorjikingsley You can send me a message on slack